test/DebugInfo/Generic/c-and-cpp-mixed.ll

```diff
@@ -70,18 +70,18 @@ target triple = "spir64-unknown-unknown"
 
 define void @foo() nounwind !dbg !5 {
 entry:
-  %puts = tail call i32 @puts(i8* getelementptr inbounds ([4 x i8], [4 x i8]* @str, i32 0, i32 0)), !dbg !23
+  %puts = tail call i32 @puts(ptr @str), !dbg !23
```

Comment:
just curious, in other tests `getelementptr` instructions are just modified, not totally removed like here. why so?